.github/workflows/ci-docker.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     types: [ opened, reopened, synchronize ]
     paths:
+      - '.github/workflows/ci-docker.yml'
```

Comment:
This is to avoid rebuilding the images unnecessarily. Saves the CPU cycles :wink: but also makes it easier to look at the CI results on PRs that don't touch Docker.